wgsl/index.bs

```diff
@@ -966,6 +1165,21 @@ The spelling of the token may be the same as an [=identifier=], but the token do
 
 Section [[#context-dependent-name-tokens]] lists all such tokens.
 
+## Diagnostic Rule Names ## {#diagnostic-rule-names}
+
+A <dfn noexport>diagnostic rule name</dfn> is a [=token=] used to name a diagnostic [=diagnostic/triggering rule=],
+in either a [=range diagnostic filter=] or a [=global diagnostic filter=].
+The spelling of the token may be the same as an [=identifier=], [=keyword=], or [=reserved word=],
```

Comment:
Q: What's the reason for allowing keywords and reserved words as diagnostic rule names? Tint currently does not support them here, and while it is surely implementable, I'm not sure there's value for authors.

Reply:
Same for reserved words too.

Reply:
will remove